package.json

```diff
@@ -143,27 +144,31 @@
     "@fortawesome/free-solid-svg-icons": "^5.15.4",
     "@fortawesome/react-fontawesome": "^0.1.16",
     "@reduxjs/toolkit": "^1.7.1",
-    "bootstrap": "^4.5.3",
+    "@tanstack/react-table": "^8.20.5",
+    "@testing-library/dom": "^10.4.0",
+    "ace-builds": "^1.36.2",
+    "bootstrap": "^4.6.2",
     "brace": "^0.11.1",
     "browserify-sign": "^4.2.2",
     "classnames": "^2.2.6",
     "compare-versions": "^4.1.2",
-    "connected-react-router": "^6.9.2",
+    "connected-react-router": "^6.9.3",
     "console-browserify": "^1.2.0",
     "core-js": "^3.20.1",
     "crypto-browserify": "^3.12.0",
     "d3": "^6.7.0",
     "d3-color": "3.1.0",
     "dagre-d3": "^0.6.4",
     "deepmerge": "^4.2.2",
-    "final-form": "4.20.4",
+    "final-form": "^4.20.4",
     "global": "^4.3.1",
     "history": "^4.10.1",
     "jquery": "^3.6.0",
     "jsonwebtoken": "^9.0.0",
     "localized-strings": "^0.2.4",
     "lodash": "^4.17.21",
     "moment": "^2.29.4",
+    "node-polyfill-webpack-plugin": "^4.0.0",
```

Comment:
Where is node-polyfill-webpack-plugin used?

Reply:
I believe it is for the Node.js core modules polyfill in Webpack since it was automatically dropped but not sure which browser polyfill we are still supporting.

Reply:
Agree - I left it in and moved it to the dev dependencies.